bloom-housing/bloom
# feat: lottery released email

This PR addresses #4161


## Description

Adds an email notification when a lottery is released to partners. It should send to everyone w access to the listing.

## How Can This Be Tested/Reviewed?

Re-seed the database to get the new translations, in other environments we will use the script runner. Add a partner user with your email and add them to a closed listing that has a lottery. Additionally, update an application to have one of your emails. As an admin, run and release the lottery. Ensure the partner receives the lottery released email. Login as the partner user, and publish the lottery. Ensure the partner receives the lottery published email and the applicant receives their version of the lottery published email.

Copy lives [here](https://docs.google.com/document/d/130OuxoQnKnukoVr_SPQq57A0JZ4vycdcS9q0EX2Gkso/edit#heading=h.l124kyiwnbpp).

![Screenshot 2024-07-17 at 11 08 19 AM](https://github.com/user-attachments/assets/2dfb857a-cbd2-47ba-98df-547fc5e21a72)
![Screenshot 2024-07-17 at 11 08 30 AM](https://github.com/user-attachments/assets/ada1c344-799e-4e49-b869-7cbba4ee4432)
![Screenshot 2024-07-17 at 11 08 44 AM](https://github.com/user-attachments/assets/679ddf02-ee92-417a-b085-ddc062762b95)


## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
